### DISCUSSION TITLE
Getting Wrong long description Font Size in work order details report

### DISCUSSION THREAD

**Discussion by eevurinaidu** [eclipse-birt/birt#1868] [Q&A] [answered]
Hello All,

Our users are trying to print the Work order details report with different font sizes. But in Final BIRT report output those font sizes are mismatching. please see the attachment.

[Maximo_Work order_ Font Size.docx](https://github.com/user-attachments/files/16691533/Maximo_Work.order_.Font.Size.docx)

Regards,
Chandra

**Comment by hvbtup** [comment]
This is for discussing BIRT, not Maximo reports.

**Reply by eevurinaidu** [reply]
I'm facing the issue in BIRT reporting only.

**Reply by merks** [reply]
You would be best to provide a sample report that someone else (who does not have access to your data source) can use to reproduce the problem you are seeing.

**Reply by eevurinaidu** [reply]
> You would be best to provide a sample report that someone else (who does not have access to your data source) can use to reproduce the problem you are seeing.

Hi Merks,

I have attached the document of issue for your reference. Please help us on the issue.

Regards,
Chandra

[BIRT reporting issue.docx](https://github.com/user-attachments/files/16746028/BIRT.reporting.issue.docx)

**Reply by merks** [reply]
I'm not so smart about how everything works; I most just help with the builds.  But the other folks are really smart and are likely to help.

**Reply by speckyspooky** [reply]
According to the hint of Ed, we would need an BIRT-rptdesign without external dependencies with your mentioned topic that we are able to reproduce the font problem.

**Reply by speckyspooky** [reply, marked as answer]
@eevurinaidu WOuld it be possible to provide a small simplified BIRT-example with your problem.
Based on your discussion I investigated the DOCX-converting #1905 and figured out a problem with the font-size.
But I wasn't able to produce your result - all was fine with font-size "small", may be I have not the correct report constellation and it was different to your case. Therefore an example would be helpful here.

**Reply by Axxpex** [reply]
I looked at a OTB woprint report.  the Control under General the Content type is HTML size 45 points (delete it). In the fetch of the dataset it is identified as HTML change it to getString, then add a Data container and choose the locdesc and match the font and size.

**Reply by speckyspooky** [reply]
Ok, I don't know at which HTML-element the font-size was set.
The size-value "45pt" is no usable value for the font-tag and it is good that you can solve it with the fixing of your HTML-snippet.

Please let me know if you should have other issues.
Currently I see here no problem with the BIRT-font style.
